### PR TITLE
complete CellIsOperators

### PR DESCRIPTION
**Pull request by jbdemonte**
## Summary

TypeScript definition of `CellIs` operators was incomplete, it missed notEqual and some others.

### CODE REVIEW

**Review by andreykrupskii — APPROVED** [review]

**Comment by andreykrupskii** [comment]
I would ask maintainers to review and merge this PR or to suggest what should be fixed (I also need this change in my company project). Thanks a lot! 😊

CC.: @Siemienik, @alubbe, @guyonroche,

File: README.md [hunk 1]
```diff
@@ -1835,12 +1835,16 @@ worksheet.addConditionalFormatting({
 
 **Cell Is Operators**
 
-| Operator    | Description |
-| ----------- | ----------- |
-| equal       | Apply format if cell value equals formula value |
-| greaterThan | Apply format if cell value is greater than formula value |
-| lessThan    | Apply format if cell value is less than formula value |
-| between     | Apply format if cell value is between two formula values (inclusive) |
+| Operator           | Description |
```
Comment (Siemienik) [inline README.md:1838]: @jbdemonte @andreykrupskii
Could you provide links to the source to confirm these changes?

Comment (Siemienik) [reply README.md:1838]: I checked, that's correct

**Review by Siemienik — APPROVED** [review]
LGTM, thank you

**Pull request closed without merging**